hermes-console/static/partials/modal/editSubscription.html

```diff
@@ -45,6 +45,17 @@ <h3 class="modal-title" ng-show="operation === 'EDIT'"><small>Edit:</small> {{su
             </div>
 
             <div class="form-group">
+                <label for="subscriptionState" class="col-md-3 control-label">Initial state</label>
+                <div class="col-md-9">
+                    <select required class="form-control" id="subscriptionState" name="subscriptionState" ng-model="subscription.state">
+                        <option value="PENDING">PENDING</option>
```

Comment:
Small change, lot's of discussion. I think the only thing we should allow (or promote via Console) is a simple checkbox "create this as suspended". This way we don't need to have all this discussion and we don't make our client to think about this choice too much.